developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Install Aptos CLI"
+id: "install-aptos-cli"
+---
+
+# Install Aptos CLI
+
+The `aptos` tool is a command line interface (CLI) for debugging, development, and node operation.
+This document describes how to install the `aptos` CLI tool. See [Use Aptos CLI](use-aptos-cli) for how to use the CLI.
+
+## Install precompiled binary (easy mode)
+
+* Navigate to the [release page](https://github.com/aptos-labs/aptos-core/releases) for Aptos CLI.
+* Download the latest release for your computer.
+* Place this at a location for you to run it e.g. `~/bin/aptos` in Linux.
+* On Linux and Mac, make this executable `chmod +x ~/bin/aptos`.
```

Comment:
We might have to note permissions from mac on allowing the executable outside of just this.

Reply:
Added this to this Linear issue to address this in a subsequent PR: https://linear.app/aptoslabs/issue/DOC-45/update-aptos-cli-doc